nginx/nginx.conf

```
@@ -0,0 +1,30 @@
+events {
+    worker_connections 1024;
+}
+http {
+  server_tokens off;
+  server {
+    listen 81;
+    root  /var/www;
+
+    location / {
+          proxy_set_header X-Forwarded-For $remote_addr;
+          proxy_set_header Host            $http_host;
+          proxy_pass http://file_hosting_server:8000/;
```

Из за этого домена прокси работает не так как надо. Здесь ты указываешь адрес твоего application приложения. У меня с такой конфигурацией не запустился твой проект. Надо так:

```suggestion
          proxy_pass http://app:8000/;
```

app - имя которое ты указал своему веб серверу в docker-compose